conda_smithy/templates/run_docker_build.sh.tmpl

```diff
@@ -69,6 +69,7 @@ DOCKER_RUN_ARGS="${CONDA_FORGE_DOCKER_RUN_ARGS}"
 if [ -z "${CI}" ]; then
     DOCKER_RUN_ARGS="-it ${DOCKER_RUN_ARGS}"
 fi
+export BUILD_WITH_MAMBABUILD="{{ 1 if build_with_mambabuild else 0 }}"
```

Comment:
So people can override it locally when using `build_locally.py`:
```suggestion
export BUILD_WITH_MAMBABUILD="${BUILD_WITH_MAMBABUILD:-{{ 1 if build_with_mambabuild else 0 }}}"
```

Reply:
@isuruf asked me to put this inside the run_docker file, so I removed it from this outer script. 

So for now one will have to add the config to the yml and rerender.

I would like to use something like shyaml or a simple grep to find the value of the option and use it more "dynamically" so that rerendering isn't necessary ... but maybe later.